packages/sync/src/config/DataSyncConfig.ts

```diff
@@ -118,3 +124,8 @@ export interface DataSyncConfig {
    */
   retryOptions?: RetryLink.Options;
 }
+
+export interface StorageOptions {
```

Comment:
We can avoid having another option. Simply we can allow people to simply pass localforage instance that is suited to their needs. By default we can use some internal localforage instance with sensible defaults.